src/app/shared/services/video.service.ts

```diff
@@ -71,4 +71,12 @@ export class VideoService {
     console.warn(`UserService: ${message}`);
   }
 
+  /** PUT: update the video on the server */
+  updateTopic(video: AppVideoItem): Observable<any> {
```

Comment:
Why do you call it updateTopic() method=??   I mean we ware inside Video service